sr/comp/mixtape/cli.py

```diff
@@ -72,7 +72,7 @@ def play_track(filename, magicq_playback, magicq_cue, generation_number, output_
             if existing_process is not None:
                 existing_process.terminate()
 
-        process = audio_controller.play(filename, output_device, trim_start)
+        process = AudioController(audio_backend).play(filename, output_device, trim_start)
```

Comment:
It would be good to keep only one `AudioController` per backend, in case more state ends up in it in the future. Perhaps putting an `lru_cache` on a function which generates them would work?